src/scales/scale.time.js

```diff
@@ -36,10 +40,94 @@ module.exports = function(Chart) {
 			},
 		},
 		ticks: {
-			autoSkip: false
+			autoSkip: false,
+			mode: 'series',   // 'linear|series'
```

Comment:
I think `auto + linear` needs to be the default based on my understanding of `series`. When the `mode` is set to series, the physical spacing between the ticks in pixels is constant. 

However, this may cause problems when the `time.min` and `time.max` settings come into play. When these settings are used, it is likely that the gap between the ticks is not constant.